.gitignore

```diff
@@ -15,5 +15,6 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+workflow_service.log
 .ruby-version
```

Comment:
you will get this file if you test locally on the bin/console script, and you don't want to commit it